### PR TITLE
chore(clerk-js): Abstract the internal form of CreateOrganization.

### PR DESCRIPTION
**Pull request by panteliselef**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.

Abstract the internal form of CreateOrganization.

This allows for other components to use it. This is preparation for a following feature that will include part of the CreateOrganization flow

Part of the new OrganizationList is the ability to create a new organization list. In order to avoid a state where no data could be displayed there, we want users to see the form that creates a new organization.
Instead of mounting the component <CreateOrganization/> inside <OrganizationList/> we want to integrate the flow direclty.

### CODE REVIEW

File: packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationPage.tsx [hunk 1]
```diff
@@ -1,175 +1,27 @@
...
-        title={title}
-        contents={<InvitationsSentMessage />}
-        sx={t => ({ minHeight: t.sizes.$60 })}
-        onFinish={completeFlow}
-      />
-    </Wizard>
+    <CreateOrganizationForm
+      skipInvitationScreen={skipInvitationScreen}
+      navigateAfterCreateOrganization={navigateAfterCreateOrganization}
+      flow={'organizationList'}
```
Comment (BRKalow) [inline packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationPage.tsx:16]: Should this be `default`?

```suggestion
      flow={'default'}
```

Comment (panteliselef) [reply packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationPage.tsx:16]: @BRKalow good catch

File: packages/clerk-js/src/ui/components/OrganizationList/CreateOrganization.tsx
Comment (BRKalow) [inline packages/clerk-js/src/ui/components/OrganizationList/CreateOrganization.tsx:1]: ❓ This file lives under `OrganizationList/`, should it actually be in `CreateOrganization/`? It seems like the intent is that we import it into the `OrganizationList` component and render it there.

Comment (panteliselef) [reply packages/clerk-js/src/ui/components/OrganizationList/CreateOrganization.tsx:1]: Probably i will move CreateOrganizationList under `/common`. That way both CreateOrganization and OrganizationList can share it, without one depending on the other

Comment (panteliselef) [reply packages/clerk-js/src/ui/components/OrganizationList/CreateOrganization.tsx:1]: this one is now resolved :)

**Review by anagstef — APPROVED** [review]
Looks good! :)

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**